lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -2044,6 +2052,15 @@ bool CommandInterpreter::HandleCommand(const char *command_line,
   m_transcript_stream << result.GetOutputData();
   m_transcript_stream << result.GetErrorData();
 
+  // Add output and error to the transcript item after splitting lines. In the
+  // future, other aspects of the command (e.g. perf) can be added, too.
+  transcript_item->AddItem(
+      "output", StructuredData::Array::SplitString(result.GetOutputData(), '\n',
+                                                   -1, false));
+  transcript_item->AddItem(
+      "error", StructuredData::Array::SplitString(result.GetErrorData(), '\n',
+                                                  -1, false));
```

Comment:
Can we time the "cmd_obj->Execute()" calls and add this as a key/value pair?

Reply:
See my rely to your other comment about I plan to do it as a separate, follow-up PR.